src/HDF5MultiDim.chpl

```diff
@@ -254,18 +256,13 @@ module HDF5MultiDim {
     * Provides the ability to store the data flat or multidimensional.
     */
     proc write_hdf_multi_msg(cmd: string, payload: string, st: borrowed SymTab): MsgTuple throws {
-        var (flat_name, shape_name, order_str, filename, dset_name, mode_str, method_str) = payload.splitMsgToTuple(7);
-        
-        var method: int;
-        try {
-            method = method_str:int;
-        } catch {
-            var errorMsg = "Could not convert method to numeric";
-            h5tLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);
-            return new MsgTuple(errorMsg, MsgType.ERROR);
-        }
+        //var (flat_name, shape_name, order_str, filename, dset_name, mode_str, method_str) = payload.splitMsgToTuple(7);
```

Comment:
couple stray commented out lines. This one and one a bit below

Reply:
Removed